### DISCUSSION TITLE
Favorites / waypoints icon size

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#333] [Q&A] [answered]
It's not that important...but...
Would it be possible to adjust the size and transparency of the favorites, regardless of the size of the icons?
2 more sliders...

**Comment by devemux86** [comment]
@Leeb64 thanks for the suggestion.

Favorite icon size could be different from other icons size.

Icon transparency is more complicated, as I did not design them with transparency in mind.
I will have to rebuild the entire collection. Also right now I don't know if it's possible for icons.

**Reply by DolomitesTom** [reply]
Separate Icon size is already useful

**Comment by devemux86** [comment]
We can have separate setting for the icon size of favorites.
Also for the icon size of waypoints.

Coming in next Cruiser.

**Comment by devemux86** [comment, marked as answer]
> We can have separate setting for the icon size of favorites.
> Also for the icon size of waypoints.

Available in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
